### PR TITLE
feat: add NVIDIA guide, re-organized repository

### PR DESCRIPTION
**Pull request by justinthelaw**
- CPU installation instruction set
- GPU installation instruction set
- Default Issue template
- Default PR template
- Changelog markdown file
- Markdownlint configurations
- README.md is now an introduction to the different install methods
- Removed CTransformers instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,409 +1,86 @@
...
-k3d cluster stop <cluster-name>
-k3d cluster start <cluster-name>
-```
+## Issues and Features
 
-## Issues and Feature Requests
+If you have issues with installation or if you believe something is missing from the installation guides then please submit an issue using the default template.
 
-For issues or questions related to this deployment guide, please submit an issue in this repository.
+## Future Improvements
```
Comment (CollectiveUnicorn) [inline README.md:77]: Lets turn these into issues and remove this section

File: INSTALL-CPU.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+```
+
+#### Kyverno Configuration
+
+As of UDS DUBBD, v0.12+, a new Kyverno policy prevents some LeapfrogAI pods from running. As we work through some refactoring to overcome these policies, the following are instructions for temporarily changing the policy from `Enforce` to `Audit`.
+
+```bash
+# open the built-in k9s CLI tool
+zarf tools monitor
+# perform the following key presses and actions:
```
Comment (CollectiveUnicorn) [inline INSTALL-CPU.md:187]: Should probably just do the kyverno config with kubectl and put the kubectl command in here

File: INSTALL-NVIDIA.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
+# LeapfrogAI Deployment Guide: NVIDIA GPUs
+
+> [!WARNING]  
+> The GPU and CPU-only installation are very similar, so if you are installing with GPU support, bring the INSTALL-CPU.md as well, since all the instructions in this markdown only point out the steps and dependencies that vary from the CPU-only instructions.
```
Comment (CollectiveUnicorn) [inline INSTALL-NVIDIA.md:4]: While I get not want to duplicate things, this requires that people go back and forth between two readme's to figure out what to do. Can we just copy over the instructions for the time being?

**Review by CollectiveUnicorn — CHANGES_REQUESTED** [review]

**Comment by CollectiveUnicorn** [comment]
What do you think about keeping the readme in one doc while just using drop downs, one for cpu and one for gpu like this: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections

**Comment by CollectiveUnicorn** [comment]
We also need to put llama-cpp instructions in there because atm there's only ctransformers

**Comment by justinthelaw** [comment]
These new instructions only include Llama-cpp. CTransformers was already removed.

**Comment by gerred** [comment]
As I finish up some of the UDS bundle work and UDS core work, do you want me to include the updated instructions in this PR or have that as a separate PR? I'm thinking rn it'll be mostly command changes and some removals, but if I can knock this out by tomorrow would love to get the updated instructions in ASAP.

**Comment by justinthelaw** [comment]
@gerred There are still some fixes I want to make on this based on Gato's feedback and for the sake of organization/prettiness.

If you pull down this branch and add a README with what you've done differently, I can merge it into the markdowns that I already have, and then test to see if I'm also able to deploy using UDS core.

**Review by CollectiveUnicorn — DISMISSED** [review]

**Review by CollectiveUnicorn — APPROVED** [review]

**Pull request merged**